sdk/monitor/Azure.ResourceManager.Monitor/src/Generated/Models/MonitorScaleCapacity.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.Monitor.Models
+{
+    /// <summary> The number of instances that can be used during this profile. </summary>
+    public partial class MonitorScaleCapacity
```

Comment:
we moved this out of custom is there any reason we had to leave the serialization.cs in custom?

Reply:
This is just a clean up - I am trying to move thing out of customization as many as possible.

This model, despite we have `int` here for every property, but in actual payload, they are strings (for instance `"1"`) and we made some customization in the serialization/deserialization to `WriteStringValue` instead of `WriteNumberValue` and use `int.TryParse` in deserialization instead of just reading strings.

Therefore those customization cannot be removed. But the customization on the model properties are not necessary therefore I took this opportunity to remove them

Reply:
Also the main purpose of this clean up is that  I want to remove the attribute `CodeGenSuppressType` for this model